sdk/dotnet/ContainerInfra/Cluster.cs

```diff
@@ -153,7 +153,7 @@ public partial class Cluster : Pulumi.CustomResource
         public Output<string> Name { get; private set; } = null!;
 
         [Output("nodeAddresses")]
-        public Output<string> NodeAddresses { get; private set; } = null!;
+        public Output<ImmutableArray<string>> NodeAddresses { get; private set; } = null!;
```

Comment:
More breaking changes similar to Linode...

Reply:
These are legit breaking changes in the TF provider - they are moving from a string to a list

https://github.com/terraform-providers/terraform-provider-openstack/commit/0db48f563e378b9feaeead4eec8c90ba7bf06c64

Reply:
Shouldn't this be a major version bump if we're going to introduce a breaking change?

Reply:
So the guidance we had used before (from Cyrus) was that it was only a major version bump if there was a risk of people's instances getting deleted. If there was a compile time change, we can minor bump - the same way that TF is doing